reactFrontenddd/src/App.jsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState, useEffect } from 'react';
+import { createRoot } from 'react-dom/client';
+import CategoryList from './components/categoryList';
+import CreateArea from './components/CreateArea';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import { useNavigate } from 'react-router-dom';
+import { useCookies } from 'react-cookie';
+import './App.css';
+import Login from './components/login';
+import APIService from './APIService'; 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faPlus} from '@fortawesome/free-solid-svg-icons';
```

Comment:
Let's re-arrange this for easy reference

```js
import React, { useState, useEffect } from 'react';
import { createRoot } from 'react-dom/client';
import { useNavigate } from 'react-router-dom';
import { useCookies } from 'react-cookie';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faPlus } from '@fortawesome/free-solid-svg-icons';

import './App.css';

import APIService from './APIService';
import CategoryList from './components/categoryList';
import CreateArea from './components/CreateArea';
import Header from './components/Header';
import Footer from './components/Footer';
import Login from './components/login';
```

Reply:
re-arranged.